packages/core/src/release.ts

```diff
@@ -611,12 +612,9 @@ export default class Release {
           hash: commit.hash,
         });
       } else if (commit.authorEmail) {
-        const author = await this.git.getUserByEmail(commit.authorEmail);
```

Comment:
`getUserByEmail` used search which is easily rate-limited, just omitting instead. It's 50/50 that the user has their email public anyway